### PR TITLE
Add pool_records and share_records to upgrade-v44 genesis

### PR DESCRIPTION
**Pull request by drklee3**

### CODE REVIEW

File: config/templates/kava/upgrade-v44/initstate/.kava/config/genesis.json [hunk 1]
```diff
@@ -896,8 +896,27 @@
         ],
         "swap_fee": "0.001500000000000000"
       },
-      "pool_records": [],
-      "share_records": []
+      "pool_records": [
```
Comment (nddeluca) [inline config/templates/kava/upgrade-v44/initstate/.kava/config/genesis.json:899]: We also have to ensure the swap module account has a balance equal to the total pool reserves - set here https://github.com/Kava-Labs/kvtool/blob/57165dc2214aeddde380725bdf9dabcaefff8fd1/config/templates/kava/upgrade-v44/initstate/.kava/config/genesis.json#L322

Comment (drklee3) [reply config/templates/kava/upgrade-v44/initstate/.kava/config/genesis.json:899]: Hmm after adding the balances and updating bank supply, I get a broken invariant: 

```
kavanode_1  | panic: invariant broken: swap: pool reserves broken invariant
kavanode_1  | pool reserves do not match module account
kavanode_1  |
kavanode_1  | 	CRITICAL please submit the following transaction:
kavanode_1  | 		 tx crisis invariant-broken swap pool-reserves
```

It runs okay without any coins set for the swap module account

Comment (nddeluca) [reply config/templates/kava/upgrade-v44/initstate/.kava/config/genesis.json:899]: ah good catch, fixed at https://github.com/Kava-Labs/kava/pull/1064

**Comment by nddeluca** [comment]
Replaced with https://github.com/Kava-Labs/kvtool/pull/74 since it sets up a few more pools

**Pull request closed without merging**